### PR TITLE
Fixes #4430, rewrote examples and parameters

### LINKED ISSUE #4430: incorrect parameter descriptions
In the parameter descriptions for maxsamples and sampleinterval the "default value" are all set to "none" and they should be set to '1'. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ba5236d2-7564-6de8-a242-f408efebf6a8
* Version Independent ID: 7fd17b39-0f00-39d5-ad46-fd143b4be0aa
* Content: [Get-Counter (Microsoft.PowerShell.Diagnostics)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.diagnostics/get-counter?view=powershell-5.1#feedback)
* Content Source: [reference/5.1/Microsoft.PowerShell.Diagnostics/Get-Counter.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Diagnostics/Get-Counter.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Get-Counter Acrolinx scores
Original version: 90
Updated version: 98

- Fixes #4430
- Fixed parameter descriptions for MaxSamples and SampleInterval
- Rewrote all the examples and restructured all the content
- Copyedits, style, links
- Fixes [AB#1555485](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1555485)

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 1]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
-The Get-Counter cmdlet gets live, real-time performance counter data directly from the performance monitoring instrumentation in Windows.
-You can use it to get performance data from the local or remote computers at the sample interval that you specify.
 
-Without parameters, a "Get-Counter" command gets counter data for a set of system counters.
+The `Get-Counter` cmdlet gets performance counter data directly from the performance monitoring
+instrumentation in the Windows family of operating systems. `Get-Counter` gets performance data from
+a local computer or remote computers.
+
+You can use the `Get-Counter` parameters to specify one or more computers, list the performance
+counter sets and the counters, set the sample intervals, and specify the maximum number of samples.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Change 

> list the performance counter sets and the counters"

to

> list the performance counter sets and the instances they contain

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 2]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
+Counter            : {\Processor(*)\% Processor Time, \Processor(*)\% User Time, ...
 ```
 
-This command gets all of the counter sets on the local computer.
+`Get-Counter` uses the **ListSet** parameter with an asterisk (`*`) to get the list of counter sets.
+The dot (`.`) in the **MachineName** column represents the local computer.
+
+### Example 2: Specify the SampleInterval and MaxSamples
+
+`Get-Counter` gets the counter data for all processors on the local computer. Data is collected at
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Change to "This examples gets..."

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 3]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
-PS C:\> Get-Counter -Counter "\Processor(_Total)\% Processor Time" -SampleInterval 2 -MaxSamples 3
+
+`Get-Counter` uses the **Counter** parameter to specify the counter path
+`\Processor(_Total)\% Processor Time`. The **SampleInterval** parameter sets a two-second interval
+to check the counter. **MaxSamples** determines that three is the maximum number of times to check
+the counter.
+
+### Example 3: Get continuous samples of a counter
+
+`Get-Counter` gets continuous samples for a counter every second. To stop the command, press
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Change to "This examples gets..."

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 4]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
+<kbd>CTRL</kbd>+<kbd>C</kbd>. To specify a longer interval between samples, se the
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Type: "se"

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 5]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
+counter sets. The **CounterSet** objects are sent down the pipeline. `Sort-Object` uses the
+**Property** parameter to specify that the objects are sorted by **CounterSetName**. The objects are
+sent down the pipeline to `Format-Table`. The **AutoSize** parameter adjusts the column widths to
+minimize truncation.
+
+The dot (`.`) in the **MachineName** column represents the local computer.
+
+### Example 5: Run a background job to get counter data
+
+`Start-Job` runs a `Get-Counter` command as a background job on the local computer. To view the
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: In this example, `Start-Job` runs...

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 6]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
-The second command uses a pipeline operator (|) to send the counter path in the $DiskReads variable to the **Get-Counter** cmdlet. The command uses the **MaxSamples** parameter to limit the output to 10 samples.
-PS C:\> $DiskReads | Get-Counter -Computer Server01, Server02 -MaxSamples 10
-```
+### Example 9: Use the PathsWithInstances property to get formatted path names
 
-These commands get the **Disk Reads/sec** counter data from the Server01 and Server02 computers.
-### Example 6
-```
-PS C:\> (Get-Counter -List PhysicalDisk).PathsWithInstances
+`Get-Counter` gets the formatted path names that include the instances for the **PhysicalDisk**
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: This example gets...

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 7]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
-PS C:\> Get-Counter -Counter $Counter -ComputerName $Servers
-```
+### Example 10: Get a single value for each counter in a counter set
 
-These commands get the value of the "**% DPC Time**" performance counter on 50 randomly select computers in the enterprise.
-### Example 8
-```
-The first command uses the **Get-Counter** cmdlet to get the counter paths. It saves them in the $MemCounters variable.
-PS C:\> $MemCounters = (Get-Counter -List Memory).Paths
+A single value is returned for each performance counter in the local computer's **Memory** counter
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: In this example, a single value...

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 8]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
+`Get-Counter` uses the **ListSet** parameter to specify the **Memory** counter set. The command is
+enclosed in parentheses so that the **Paths** property returns each path as a string. The paths are
+stored in the `$MemCounters` variable. `Get-Counter` uses the **Counter** parameter to specify the
+counter paths in the `$MemCounters` variable.
+
+### Example 11: Display an object's property values
 
-The third command uses the Format-List cmdlet to display all the properties of the **CounterSamples** property of the sample set object as a list.
-PS C:\> $Data.CounterSamples | Format-List -Property *
+The property values in the **PerformanceCounterSample** object represent each data sample. You can
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: The property values in the **PerformanceCounterSample** object represent each data sample. In this example we use the properties of the **CounterSamples** object to examine, select, sort, and group the data.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 9]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
-```
+The counter path is stored in the `$Counter` variable. `Get-Counter` gets one sample of the counter
+values and stores the results in the `$Data` variable. The `$Data` variable uses the
+**CounterSamples** property to get the object's properties. The object is sent down the pipeline to
+`Format-List`. The **Property** parameter uses an asterisk (`*`) wildcard to select all the
+properties.
 
-The command runs a **Get-Counter** command as background job.
-For more information, see Start-Job.
+### Example 12: Use the CounterSamples property to display counters
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Example 11 and 12 are really duplicates. Remove Example 12 and renumber remaining.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 10]
```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
...
 ### -ComputerName
-Gets data from the specified computers.
-Type the NetBIOS name, an Internet Protocol (IP) address, or the fully qualified domain names of the computers.
-The default value is the local computer.
 
-Note:  Get-Counter does not rely on Windows PowerShell remoting.
-You can use the ComputerName parameter of Get-Counter even if your computer is not configured for remoting in Windows PowerShell.
+Specifies one computer name or a comma-separated array of **remote** computer names. Use the NetBIOS
+name, an IP address, or the computer's fully qualified domain name. **ComputerName** accepts strings
+from variables or values from a text file.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Values from a text file? How? Strike the whole sentence. Not needed.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md [hunk 11]
```diff
@@ -301,123 +507,167 @@ Aliases:
...
+
+`Get-Counter` accepts pipeline input for counter paths and counter set names.
+
 ## OUTPUTS
 
-### Microsoft.PowerShell.Commands.GetCounter.CounterSet, Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSampleSet, Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSample
-The ListSet parameter gets Microsoft.PowerShell.Commands.GetCounter.CounterSet objects.
-The Counter parameter gets Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSampleSet objects.
-Each counter value is a Microsoft.PowerShell.Commands.GetCounter.PerformanceCounterSample object.
+### Output objects
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: The H3 here needs to list the type names of the output object. So please restore the previous text.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 4 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: Should be **use**. resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 5 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 7 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 8 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 9 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 10 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 11 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

File: reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md (hunk 6 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md]: resolved.

**Pull request merged**